promql/parse.go

```diff
@@ -112,6 +120,105 @@ func (p *parser) parseExpr() (expr Expr, err error) {
 	return
 }
 
+// sequenceValue is a omittable value in a sequence of time series values.
+type sequenceValue struct {
+	Value   clientmodel.SampleValue
```

Comment:
Hum, I've never seen this before except when it's actually needed (like for JSON marshaling). Looks odd to me :)